### PR TITLE
Add Polish

### PR DESCRIPTION
**Pull request by mbiesiad**
Hi. First of all, great project! Here: https://github.com/mbiesiad/tldr/tree/master/pages.pl/common
are translated pages/common into Polish 🎉 

Best greetings

### CODE REVIEW

File: pages.pl/common/7z.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> Archiwizator plików o wysokim stopniu kompresji.
+> Więcej informacji: <https://www.7-zip.org/>.
+
+- Zarchiwizuj plik lub katalog:
+
+`7z a {{archived.7z}} {{path/to/file_or_directory}}`
```
Comment (zdroid) [inline pages.pl/common/7z.md:8]: Here and pretty much in all other files, you should translate the text inside `{{ }}`, too, unless it's a list of concrete options like `{{zip|gzip|bzip2|tar}}`.

Example: `archived` from `{{archived.7z}}` should be translated, `{{path/to/file_or_directory}}` should be translated completely.

Comment (mbiesiad) [reply pages.pl/common/7z.md:8]: Now I see. Thanks for your note! I will fix it

Comment (mbiesiad) [reply pages.pl/common/7z.md]: done

**Comment by zdroid** [comment]
Based on my limited understanding of Polish, this is all good. :laughing:

Thanks for the great work. @sbrl @agnivade is this PR to be squashed or rebased?

**Comment by agnivade** [comment]
The author has taken the effort of creating individual commits for separate pages, so let's rebase this. :+1:

**Comment by alufers** [comment]
As a native Polish speaker, I have gone through a few pages and they look like they are spot on. They don't look like machine translation, so this is very helpful.

Good job! :tada:

**Comment by agnivade** [comment]
Thanks for the input @alufers !

**Comment by bl-ue** [comment]
I know this is really old, but I just came upon this and was so amazed 🤩 by your perseverance @mbiesiad that I decided to go out of the ordinary to compliment and thank you for your incredible work!  26 pages translated?!?!  Amazing!!! 😍

Thank you so much for helping to bring tldr-pages to the Poles!! 😄 :heart: 😆

**Comment by sbrl** [comment]
@bl-ue we've got a number of talented translators. I think @Proscream did a number of PRs translating to Chinese, in which there were over 50 pages per PR! O.o

**Comment by bl-ue** [comment]
~Chinese~ **Korean**

I know! It's great.

**Pull request merged**